DigiMeSDK/Core/Classes/Entities/CATimeRange.m

```diff
@@ -0,0 +1,68 @@
+//
+//  CATimeRange.m
+//  DigiMeSDK
+//
+//  Created on 27/11/2018.
+//
+
+#import "CATimeRange.h"
+
+@interface CATimeRange()
+
+@property (nonatomic, strong, nullable, readwrite) NSDate *from;
+@property (nonatomic, strong, nullable, readwrite) NSDate *to;
+@property (nonatomic, strong, nullable, readwrite) NSString *last;
+
+@end
+
+@implementation CATimeRange
+
++ (CATimeRange *)from:(NSDate *)from
+{
+    CATimeRange *range = [CATimeRange new];
+    range.from = from;
+    
+    return range;
+}
+
++ (CATimeRange *)from:(NSDate *)from to:(NSDate *)to
+{
+    CATimeRange *range = [CATimeRange new];
```

Comment:
Should we validate that `from` is before `to`?

Reply:
We might as well!